src/DI/Extension.php

```diff
@@ -6,7 +6,7 @@ final class Extension extends \Nette\DI\CompilerExtension
 {
 
 	/**
-	 * @var string
+	 * @var class-string
```

Comment:
Je to nějaká fíčura phpstanu to takto zapsat nebo to je typo?

Reply:
Fíčr: https://medium.com/@ondrejmirtes/generics-in-php-using-phpdocs-14e7301953